tests/pages/pay-page.js

```diff
@@ -1,16 +1,38 @@
-import { clickable, collection, create, text, triggerable, visitable } from 'ember-cli-page-object';
+import { clickable, collection, create, text, visitable } from 'ember-cli-page-object';
 import AccountDropdown from 'codecrafters-frontend/tests/pages/components/account-dropdown';
 import Header from 'codecrafters-frontend/tests/pages/components/header';
 
 export default create({
   accountDropdown: AccountDropdown,
-  clickOnApplyRegionalDiscountButton: clickable('[data-test-apply-regional-discount-button]'),
-  clickOnExtraInvoiceDetailsToggle: clickable('[data-test-extra-invoice-details-toggle]'),
-  clickOnProceedToCheckoutButton: clickable('[data-test-proceed-to-checkout-button]'),
-  clickOnStartPaymentButtonForMonthlyPlan: clickable('[data-test-monthly-pricing-card] [data-test-start-payment-button]'),
-  clickOnStartPaymentButtonForYearlyPlan: clickable('[data-test-yearly-pricing-card] [data-test-start-payment-button]'),
+
+  chooseMembershipPlanModal: {
+    scope: '[data-test-choose-membership-plan-modal]',
+    clickOnChoosePlanButton: clickable('[data-test-choose-plan-button]'),
+    clickOnExtraInvoiceDetailsToggle: clickable('[data-test-extra-invoice-details-toggle]'),
```

Comment:
Same note as previously added - Let's stick to convention and not add spaces unless there's a nested block that needs spaces around it (all single-line properties don't need spaces around them)